js/components/Star/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  TouchableOpacity,
+  Image,
+  StyleSheet,
+} from 'react-native';
+
+export default class Star extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: false,
+    };
+
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle() {
+    this.setState({ value: !this.state.value });
+  }
+
+  render() {
+    const img = this.state.value ?
```

Comment:
Does it work this way? I tried a few days ago and it didn't work. `require()` has to be static.